hypertrace-core-graphql-attribute-scope/src/main/java/org/hypertrace/core/graphql/atttributes/scopes/HypertraceCoreAttributeScope.java

```diff
@@ -4,7 +4,8 @@
 
 enum HypertraceCoreAttributeScope implements AttributeScope {
   TRACE(HypertraceCoreAttributeScopeString.TRACE),
-  SPAN(HypertraceCoreAttributeScopeString.SPAN);
+  SPAN(HypertraceCoreAttributeScopeString.SPAN),
+  LOG_EVENT(HypertraceCoreAttributeScopeString.LOG_EVENT);
```

Comment:
This shouldn't be needed, the enum scopes are deprecated and only retained for backwards compat

Reply:
was just trying some random things, removed